src/main/java/com/sysdig/jenkins/plugins/sysdig/Util.java

```diff
@@ -15,11 +15,23 @@
 */
 package com.sysdig.jenkins.plugins.sysdig;
 
-import com.google.common.base.Splitter;
-import java.util.regex.Pattern;
+import java.nio.file.Paths;
+import java.nio.file.InvalidPathException;
 
 public class Util {
 
   public enum GATE_ACTION {PASS, FAIL}
   public enum GATE_SUMMARY_COLUMN {Repo_Tag, Stop_Actions, Warn_Actions, Go_Actions, Final_Action}
+
+  public static boolean isValidLocalPath(String path) {
+    try {
+      Paths.get(path);
+    } catch (InvalidPathException | NullPointerException ex) {
+      return false;
+    }
+    if (path.length()>0 && path.substring(0, 1).equals("/")){
```

Comment:
I guess we only support absolute paths, but that should be ok! Maybe using path.startswith("/") would be slightly shorter?

Reply:
:+1:

Reply:
Nice catch Alvaro! Fortunately right now docker only supports absolute path https://github.com/docker/cli/issues/1203#issuecomment-981757086  :)